src/bidiMapper/domains/network/NetworkProcessor.ts

```diff
@@ -82,23 +82,54 @@ export class NetworkProcessor {
       NetworkProcessor.parseUrlString(params.url);
     }
 
-    const {url, method} = params;
+    const {url, method, headers} = params;
     // TODO: Set / expand.
-    // ; headers
     // ; cookies
     // ; body
 
+    const requestHeaders: Protocol.Fetch.HeaderEntry[] | undefined =
+      cdpFetchHeadersFromBidiNetworkHeaders(headers);
+
     await this.#networkStorage
       .getRequest(networkId)
-      ?.continueRequest(fetchId, url, method);
+      ?.continueRequest(fetchId, url, method, requestHeaders);
 
     this.#networkStorage.removeBlockedRequest(networkId);
 
     return {};
   }
 
-  continueResponse(_params: Network.ContinueResponseParameters): EmptyResult {
-    throw new UnknownCommandException('Not implemented yet.');
+  async continueResponse(
+    params: Network.ContinueResponseParameters
+  ): Promise<EmptyResult> {
+    const networkId = params.request;
+    const {request: fetchId, phase} = this.#getBlockedRequest(networkId);
+
+    if (phase === Network.InterceptPhase.BeforeRequestSent) {
+      throw new InvalidArgumentException(
+        `Blocked request for network id '${networkId}' is in 'BeforeRequestSent' phase`
+      );
+    }
+
+    const {statusCode, reasonPhrase, headers} = params;
+
+    // TODO: Step 6
```

Comment:
No need. In the spec, the `update the response` is used for both `continueResponse` and `continueRequest`. The assertion is there only for readibility.